fdbclient/ServerKnobs.cpp

```diff
@@ -179,6 +179,8 @@ void ServerKnobs::initialize(Randomize randomize, ClientKnobs* clientKnobs, IsSi
 	init( MAX_DEST_CPU_PERCENT, 		  					   100.0 );
 	init( DD_TEAM_PIVOT_UPDATE_DELAY,                            5.0 );
 
+	init( ALLOW_LARGE_SHARD,                                    true );
```

Comment:
Maybe buggify the value to test when it's false? Can we default this to `false`?

Reply:
Thanks for catching it, done.